README.md

```
@@ -32,16 +33,23 @@
+    - [StrongBox.kt](src/main/kotlin/day10/StrongBox.kt) - StrongBox Class
+    - [Word.kt](src/main/kotlin/day10/Word.kt) - Word Class
+    - [StrongBoxTest.kt](src/test/kotlin/day10/StrongBoxTest.kt) - StrongBox Test Code
+    - [WordTest.kt](src/test/kotlin/day10/WordTest.kt) - Word Test Code
 
 ## 📚 주제별 정리
 
 - [Git](src/docs/topics/git.md)
 - Kotlin
     - [Class](src/docs/topics/class.md)
     - [Collection](src/docs/topics/kotlin-collection.md)
+    - [Equals](src/docs/topics/equals.md)
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)